app/models/sidebar_section.rb

```diff
@@ -24,6 +24,7 @@ class SidebarSection < ActiveRecord::Base
             }
 
   scope :public_sections, -> { where("public") }
+  enum :section_type, { community: 0 }, scopes: false, suffix: true
```

Comment:
Just realised this, do we need some sort of validation on the model to ensure this is unique?

Reply:
Fair point, added unique index. For all other public/private sections, this is null so index will be satisfied